charts/portal/templates/jobs/db-upgrade-job.yaml

```diff
@@ -25,7 +25,7 @@ spec:
       containers:
         - name: db-upgrade
           image: "{{ .Values.global.portalRepository }}{{ .Values.image.dbUpgrade }}"
-          imagePullPolicy: IfNotPresent
+          imagePullPolicy: {{ index .Values "db-upgrade-portal.image.pullPolicy" }}
```

Comment:
apply to all jobs, including rbac-upgrade and cert-update.
also for the variable name, i suggest just one variable for all three jobs under https://github.com/CAAPIM/apim-charts/blob/stable/charts/portal/values.yaml#L646
so `jobs.image.pullPolicy`
also update https://github.com/CAAPIM/apim-charts/blob/stable/charts/portal/README.md#configuration for new variables